src/patternfly/layouts/Gallery/styles.scss

```diff
@@ -0,0 +1,7 @@
+.pf-l-gallery {
+  display: grid;
+  grid-gap: 1rem; //revisit this value
```

Comment:
This should be a variable, something like "gutter" to use on different layouts and it should be themable.

Reply:
Correct, but we originally said that layouts wouldn't pull in global variables.

Reply:
Do you mean you just want a the component variable for now?

Reply:
yeap, we did say that. But we need a variable anyway. maybe this is an exception

Reply:
yes, a component type variable that calls a global variable

Reply:
If I remove the grid-gap all together, we can shift responsibility to the padding of the child, and then an exception wouldn't be necessary.

If we take that route, we could also circle back later after we've had time to think through introduction of global variables and how that will affect the system as a whole.

What do you think?